pkg/engine/apicall/apiCall.go

```diff
@@ -101,7 +101,9 @@ func (a *apiCall) executeK8sAPICall(ctx context.Context, path string, method kyv
 
 	jsonData, err := a.client.RawAbsPath(ctx, path, string(method), requestData)
 	if err != nil {
-		return nil, fmt.Errorf("failed to %v resource with raw url\n: %s: %v", method, path, err)
+		errMsg := fmt.Sprintf("failed to %v resource with raw url\n: %s: %v", method, path, err)
+		a.logger.Error(err, errMsg, "name", a.entry.Name)
```

Comment:
i would prefer structured login TBH.
```suggestion
		a.logger.Error(err, "failed to call service", "name", a.entry.Name, "method", method, "path", path)
```